### DISCUSSION TITLE
How do I change Uploads folder to a different folder?

### DISCUSSION THREAD

**Discussion by tablifiedapps** [OctoFarm/OctoFarm#1337] [Q&A] [answered]
I have multiple instances of octoprint running off my Linux server. I installed them via octoprintdeploy. I would like each instance to access a specific folder on my internal hdd for gcode files. Where can I change the path?

**Comment by NotExpectedYet** [comment]
> I have multiple instances of octoprint running off my Linux server. I installed them via octoprintdeploy. I would like each instance to access a specific folder on my internal hdd for gcode files. Where can I change the path?

This isn't a question about OctoFarm it's about OctoPrint / OctoPrintDeploy and would be better served asking there.

**Comment by NotExpectedYet** [comment, marked as answer]
> I have multiple instances of octoprint running off my Linux server. I installed them via octoprintdeploy. I would like each instance to access a specific folder on my internal hdd for gcode files. Where can I change the path?

https://docs.octoprint.org/en/master/configuration/config_yaml.html#folder

This may help

**Comment by tablifiedapps** [comment]
Thanks!
